lang/src/org/partiql/lang/eval/NaturalExprValueComparators.kt

```diff
@@ -55,9 +55,17 @@ import org.partiql.lang.util.isZero
  *  * `BAG` values come finally (except with [NullOrder.NULLS_LAST]), and their values
  *    compare lexicographically based on the *sorted* child elements.
  */
-enum class NaturalExprValueComparators(private val nullOrder: NullOrder) : Comparator<ExprValue> {
-    NULLS_FIRST(NullOrder.FIRST),
-    NULLS_LAST(NullOrder.LAST);
+enum class NaturalExprValueComparators(private val order: Order, private val nullOrder: NullOrder) : Comparator<ExprValue> {
+    NULLS_FIRST_ASC(Order.ASC, NullOrder.FIRST),
+    NULLS_FIRST_DESC(Order.DESC, NullOrder.FIRST),
+    NULLS_LAST_ASC(Order.ASC, NullOrder.LAST),
+    NULLS_LAST_DESC(Order.DESC, NullOrder.LAST);
+
+    /** Whether or not null values come first or last. */
```

Comment:
typo: doc comment should talk about the ordering spec not null order spec

Reply:
fixed with new commit